conman/redirects/tests/test_views.py

```diff
@@ -43,7 +43,8 @@ def test_access_redirect(self):
         """Accessing a RouteRedirect's url redirects to its target's url."""
         target = ChildRouteFactory.create()
         route = ChildRouteRedirectFactory.create(target=target)
-        response = self.client.get(route.url)
+        with ignore_django20_deprecation():
+            response = self.client.get(route.url)
```

Comment:
This complains about the [`MIDDLEWARE` setting](https://docs.djangoproject.com/en/1.10/topics/http/middleware/#upgrading-middleware) that replaces `MIDDLEWARE_CLASSES`.

As far as I can tell, we can't switch until we drop support for 1.8 and 1.9.